tests/testapp/urls.py

```diff
@@ -8,6 +8,7 @@
 router.register(r'df-users', views.DFUserViewSet, basename='df-users')
 router.register(r'ff-users', views.FilterFieldsUserViewSet, basename='ff-users')
 router.register(r'ffcomplex-users', views.ComplexFilterFieldsUserViewSet, basename='ffcomplex-users')
+router.register(r'ffjsoncomplex-users', views.ComplexJsonFilterFieldsUserViewSet, basename='ffcomplex-users')
```

Comment:
PEP-8 nitpick: `ComplexJSONFilterFieldsUserViewSet`

Reply:
There are some other places with `Json`, but I've updated in this PR. Thanks for the suggestion!